src/NJsonSchema.Tests/Validation/FormatDateTimeTests.cs

```diff
@@ -66,7 +66,7 @@ public void When_format_date_time_with_iso8601_then_validation_succeeds()
             schema.Type = JsonObjectType.String;
             schema.Format = JsonFormatStrings.DateTime;
 
-            var token = new JValue("2015-01-25T15:43:30Z");
+            var token = new JValue("2015-01-25T15:43:30.1234567Z");

```

Comment:
Please add a new test case for the new format so that the current one is still checked and we dont have regressions

Reply:
Added